lib/rspec/mocks/any_instance/stub_chain.rb

```diff
@@ -38,7 +37,7 @@ def invocation_order
         end
 
         def verify_invocation_order(rspec_method_name, *_args, &_block)
-          return if invocation_order[rspec_method_name].include?(last_message)
+          return if invocation_order.fetch(rspec_method_name, [nil]).include?(last_message)
```

Comment:
No because that should never happen, and I can't think of any remaining scenarios where it would happen, but after removing the "empty" values from the hashes this change was necessary to turn the build back to green, does that suffice?